### DISCUSSION TITLE
auth-v3.1.1

### DISCUSSION THREAD

**Discussion by prateekmedia** [ente-io/ente#2681] [General]
## What's Changed
* [auth] don't use privacy screen for desktop by @prateekmedia in https://github.com/ente-io/ente/pull/2680

**Full Changelog**: https://github.com/ente-io/ente/compare/auth-v3.1.0...auth-v3.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ente-io/ente/releases/tag/auth-v3.1.1'>auth-v3.1.1</a>.</em>

**Comment by euphorical** [comment]
The ente-auth-v3.1.1-installer.exe" file is getting flagged by Windows as potentially harmful, & Virustotal shows "W32.AIDetectMalware" & "Win/grayware_confidence_60% (D)".
https://www.virustotal.com/gui/file/cbd462143c77b310e5a8a9b578127b6475bf36cc4bc9249d7a053d680d5db4a1

**Reply by prateekmedia** [reply]
It is a false positive, see:
https://www.reddit.com/r/antivirus/comments/14zijs6/hi_is_this_a_virus/
https://www.reddit.com/r/antivirus/comments/wbbfrb/what_does_wingrayware_confidence_70d_mean/
